Octokit/Models/Response/PullRequest.cs

```diff
@@ -211,6 +211,11 @@ public bool Merged
         /// </summary>
         public bool Locked { get; protected set; }
 
+        /// <summary>
+        /// Users requested for review
+        /// </summary>
+        public IReadOnlyList<User> RequestedReviewers { get; protected set; }
```

Comment:
Dont forget to add this new property to the `ctor` of the response class as well